src/unshadow.c

```diff
@@ -97,17 +97,36 @@ static void read_file(char *name, void (*process_line)(char *line))
 	if (fclose(file)) pexit("fclose");
 }
 
+static int spaceTab(char c) {
+	if (' ' == c) {
+		return 1;
+	}
+	if ('\t' == c) {
+		return 1;
+	}
+
```

Comment:
This is against our coding style. We never `if (constant == variable)` but instead `if (variable == constant)`. More instances of same problem below

Reply:
Fixed in the updated commit.